### ISSUE TITLE
Incompatible with storyshots

### ISSUE THREAD

**Issue by bjankord** [storybooks/storybook-addon-a11y#17]
We have a project we are using [storyshots](https://github.com/storybooks/storybook/tree/master/addons/storyshots) with storybook-addon-a11y. While we can run storybook in the browser without issue, when we run the tests based on the storyshots, we are getting multiple `Invariant Violation: getNodeFromInstance: Invalid argument.` errors.

Doing a quick google search on this error message led me to https://github.com/facebook/react/issues/8324

It makes me think the [use of findDomNode here](https://github.com/storybooks/storybook-addon-a11y/blob/cb8f8719832a1cdab773bb54d48ce0396d0ee7f0/src/components/WrapStory.js#L15) is what is causing compatibility issues with storyshots.

**Comment by bjankord** [comment]
@zinserjan Are you able to run this with storyshots, I saw your [commit](https://github.com/storybooks/storybook-addon-a11y/commit/96a820aec52ca3766f48403c8a25dbb130fb94e2) for updating WrapStory.js from using refs to findDomNode was to help with testing against the original html structure with storyshot’s snapshot testing.

**Comment by bjankord** [comment]
I tried downgrading the version of storybook-addon-a11y to v3.0.0 which uses refs instead of findDOMNode and snapshot tests worked, though they had the extra span element which I believe is was @zinserjan was working to remove in his commit with switching to using findDOMNode.

**Comment by ndelangen** [comment]
@bjankord Is there some way you could help us with bringing back compatibility?

**Comment by bjankord** [comment]
I'd really like to, I think this is a valuable addon for storybook, but at this time I don't foresee having the capacity to achieve this.

**Comment by ndelangen** [comment]
I'll be achieving this repo soon, after merging this into the monorepo.

Whilst doing this I was actually able to run storyshots just fine, maybe I fixed the problem?
https://github.com/storybooks/storybook/pull/2292/commits/862d4cf2a76bc029ae8243225fdf77e4b4a0e307

**Comment by ndelangen** [comment]
FYI I have merged this repo into the monorepo!

Please do not open any issues or PR's directly on this repo, but instead open them here:
https://github.com/storybooks/storybook
